tests/test_main.py

```diff
@@ -89,6 +92,48 @@ async def test_list_properties(connection: aiosqlite.Connection):
     assert results
 
 
+@pytest.mark.asyncio
+async def test_multiple_categories(connection: aiosqlite.Connection):
+    """
+    Test that when given multiple categories for a node
+    we return all of them
+    """
+    await add_data(
+        connection,
+        data="""
+            MONDO:0005148(( category biolink:Disease ))
+            MONDO:0005148<-- predicate biolink:treats --CHEBI:6801
+            CHEBI:6801(( category biolink:ChemicalSubstance ))
+            CHEBI:6801(( category biolink:Drug ))
```

Comment:
I was expecting something like `CHEBI:6801(( category biolink:ChemicalSubstance category biolink:Drug ))`. I don't know which is more readable. Or is there a third alternative?